src/questions/pancakeswap.yaml

```diff
@@ -0,0 +1,171 @@
+- uuid: a85b1753-f2da-4172-a420-50300ffe1c73
+  type: SingleChoice
+  content: What is Pancakeswap?
```

Comment:
Since pancake swap has a lot of products, it now quite more than DEX. So we can ask, select the products from pancake swap